src/libs/actions/User.js

```diff
@@ -268,6 +267,45 @@ function isBlockedFromConcierge(blockedFromConcierge) {
 }
 
 /**
+ * Adds a paypal.me address for the user
+ *
+ * @param {String} address
+ */
+function addPaypalMeAddress(address) {
+    const optimisticData = [
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: ONYXKEYS.NVP_PAYPAL_ME_ADDRESS,
+            value: address,
+        },
+    ];
+    API.write('AddPaypalMeAddress', {
+        value: address,
+    }, {optimisticData});
+}
+
+/**
+ * Deletes a paypal.me address for the user
+ *
+ */
+function deletePaypalMeAddress() {
+    const optimisticData = [
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: ONYXKEYS.NVP_PAYPAL_ME_ADDRESS,
+            value: '',
+        },
+    ];
+    API.write('DeletePaypalMeAddress', {}, {optimisticData});
```

Comment:
even if there is no value it feels weird to have this in a single line, unless linter asks it to be like this I would have in multiple lines